src/install/mod.rs

```diff
@@ -37,7 +33,13 @@ fn download_prebuilt(
         )),
     };
 
-    let binaries = &[tool_name];
+    let binary_name = if cfg!(target_os = "windows") {
+        format!("{}.exe", tool_name)
+    } else {
+        tool_name.to_owned()
```

Comment:
Here I had to make a copy of it because of the format above, any way to make it better?